lib/configuration.ts

```diff
@@ -0,0 +1,56 @@
+import { getEnvVars, setEnvVars } from "./environment";
+import { createClient } from "./graphql";
+import prisma from "./prisma";
+
+export interface Configuration {
+  domain: string;
+  token: string;
+}
+
+const MT_MODE = true;
```

Comment:
I refer to comment below, ofc - this is POC - so for the actual version, I'd rather avoid "branching" behavior and move the only difference here to persistence level.

Then we have clear path (if I understand it) - the difference between ST an MT is app capable of persisting multiple Saleor connections.

The question is, are there any other differences we must consider? I hope we can unify them

Reply:
A new proposal introduces the app conf object, where developer can specify which APL needs to be used 🎉